mlflow/R/mlflow/tests/testthat/test-tracking-experiments.R

```diff
@@ -3,27 +3,27 @@ context("Tracking - Experiments")
 test_that("mlflow_create/get_experiment() basic functionality (fluent)", {
   mlflow_clear_test_dir("mlruns")
 
-  experiment_1 <- mlflow_create_experiment("exp_name", "art_loc")
```

Comment:
Change tests since `mlflow_create_experiment` now returns the string ID instead of an experiment object